### PR TITLE
Add spinner.warn()

### PR DESCRIPTION
**Pull request by michael-ciniawsky**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 language: node_js
 node_js:
-  - '6'
-  - '4'
+  - 'node'
```
Comment (SamVerschueren) [inline .travis.yml]: Remove `node` as test version

File: .travis.yml [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
+  - 6
+  - 4
```
Comment (sindresorhus) [inline .travis.yml]: Don't do unrelated changes.

Comment (michael-ciniawsky) [inline .travis.yml]: kk

File: .travis.yml (hunk 1 above)
Comment (michael-ciniawsky) [inline .travis.yml]: kk

**Comment by michael-ciniawsky** [comment]
@sindresorhus @SamVerschueren spinner.info() 🙃 ? I remove `node` from travis as requested and add a test for warnings with text changes.

File: .travis.yml (hunk 1 above)
Comment (michael-ciniawsky) [inline .travis.yml]: done

File: .travis.yml (hunk 2 above)
Comment (michael-ciniawsky) [inline .travis.yml]: done

**Comment by sindresorhus** [comment]
Thanks :)

> spinner.info() 🙃 ?

Sure. PR welcome :)

**Comment by michael-ciniawsky** [comment]
@sindresorhus Thx 😛 

> Sure. PR welcome.

kk
